### ISSUE TITLE
missing name qualification leads to MethodError

### ISSUE THREAD

**Issue by rapus95** [Gnimuc/Sodium.jl#3]
https://github.com/Gnimuc/Sodium.jl/blob/a85c4c96a3c75e04e4048be30589ff1c86a57e9b/src/crypto.jl#L5
This obviously cannot work 😁 You either have to give the helper function a new more idiomatic name  (which doesn't need to have the same name as the function it makes more convenient) or qualify the access into LibSodium.
And I also got to thinking about whether String is appropriate as return type... Maybe rather convert it to a byte array.
@Gnimuc

**Comment by rapus95** [comment]
We probably should add a test for it 🤔

**Comment by rapus95** [comment]
```julia
using Base64

function seal(msg, pk)
    len = sizeof(msg)
    ciphertext = Vector{Cuchar}(undef, crypto_box_SEALBYTES + len)
    LibSodium.crypto_box_seal(ciphertext, msg, len, pk)
    return base64encode(ciphertext)
end
```
Calling it simply `seal` seems to be a good bet as no other language bindings showed there: https://developer.github.com/v3/actions/secrets/#create-or-update-a-secret-for-a-repository
use the full name. Calling it `seal` would follow the js binding `tweetsodium`.

**Comment by Gnimuc** [comment]
I've just invited you as a collaborator so you can make these changes directly ;).

**Comment by rapus95** [comment]
Are you fine with releasing a proper minor bump after it? As we added new features? (Well, the first feature actually since the others are hidden inside the non-exported LibSodium 😄)

**Comment by Gnimuc** [comment]
yep, we should follow semantic versioning.

**Comment by rapus95** [comment]
Forgot to add tests and found the reason why those didn't work. Will need to add another encoding/decoding. The PK will come as base64 encoded (from github) and thus we cannot simply put it as is into the function.

**Comment by rapus95** [comment]
fixed by ef9723f1fe
